projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -583,3 +583,29 @@ function wpcom_add_scheduled_updates_menu() {
 	);
 }
 add_action( 'admin_menu', 'wpcom_add_scheduled_updates_menu' );
+
+/**
+ * Add the Plugins menu item to the admin menu on simple sites.
+ */
+function wpcom_add_plugins_menu() {
+
+	if ( ! defined( 'IS_WPCOM' ) || ! IS_WPCOM ) {
+		return;
+	}
+
+	if ( function_exists( 'wpcom_is_nav_redesign_enabled' ) && wpcom_is_nav_redesign_enabled() ) {
+		$domain              = wp_parse_url( home_url(), PHP_URL_HOST );
+		$can_install_plugins = function_exists( 'wpcom_site_has_feature' ) && wpcom_site_has_feature( WPCOM_Features::INSTALL_PLUGINS );
+
+		add_menu_page(
+			__( 'Plugins', 'jetpack-mu-wpcom' ),
```

Comment:
Ideally we should escape this since it's going to be injected as a `title` ~attribute~ tag:
```suggestion
			esc_attr__( 'Plugins', 'jetpack-mu-wpcom' ),
```

Reply:
oh, thanks for catching that. I assumed that WP would just escape that stuff automatically rather than every person monkeying with the menu having to do it.

I guess the next parameter should be `esc_html`'d in that case.

Reply:
If it's a tag then it should be `esc_html` not `esc_attr`....and yet jetpack is full of people either not doing anything with this, or using `esc_attr__` 🤔

Reply:
> I assumed that WP would just escape that stuff automatically rather than every person monkeying with the menu having to do it.

Yeah, I thought the same, but apparently that's the intended behavior. See https://core.trac.wordpress.org/ticket/56655

And good point about escaping both the 1st and the 2nd parameters with `esc_html` rather than `esc_attr`(in the linked ticket it was suggested to use `wp_kses` too).

Said that, I think I agree with the ultimate reason why the ticket was closed as "won't fix": we only need to escape user-generated content, not the one generated by plugin authors (I guess translators also count as authors!).